src/controllers/busControllers.js

```diff
@@ -0,0 +1,72 @@
+import models from '../database/models';
+
+const createBus = async (req, res) => {
+  try {
+    const bus = await models.Bus.create(req.body);
+    return res.status(201).json({ status: 201, message: 'Bus created successfully', bus });
+  } catch (error) {
+    return res.status(500).json({ error: error.message });
+  }
```

Comment:
I don't see where you handle duplication here. When the user tries to create the same resource twice

Reply:
It is handled now, and Thank you for reminding me that